code/modules/clothing/suits/shirt.dm

```diff
@@ -4,6 +4,18 @@
 	icon_state = "ianshirt"
 	inhand_icon_state = "ianshirt"
 	species_exception = list(/datum/species/golem)
+	///How many times has this shirt been washed? (In an ideal world this is just the determinant of the transform matrix.)
+	var/wash_count = 0
+
+/obj/item/clothing/suit/ianshirt/machine_wash(obj/machinery/washing_machine/washer)
+	. = ..()
+	if(wash_count <= 5)
+		transform *= TRANSFORM_USING_VARIABLE(0.8, 1)
+		washer.visible_message(span_notice("\the [src] appears to have shrunk down after a long machine washing."))
+		wash_count += 1
+	else
+		washer.visible_message(span_notice("\the [src] appears to have imploded due to repeat washing. Tiny strands of energy from the wash-dimension linger."))
```

Comment:
use balloon_alert here and try to reduce the size.
Balloon alerts are better because they appear visually and not in chat.

Reply:
example
```suggestion
		washer.balloon_alert_to_viewers("[src] implodes into tiny strands!")
```